quarkchain/cluster/shard_state.py

```diff
@@ -1995,4 +2015,29 @@ def get_total_balance(
         limit: int,
         starter: Optional[bytes] = None,
     ) -> Tuple[int, bytes]:
-        return 42, b""
+        """
+        The starter address should be exclusive in the iteration.
+        """
+        evm_state = self._get_evm_state_from_hash(block_hash)
+        if not evm_state:
+            # TBD, may do hash validation before
+            return 0, b""
```

Comment:
how about return None and let slave return error response?

Reply:
sure~